### ISSUE TITLE
issue with latest release of djangorestframework

### ISSUE THREAD

**Issue by kamedodji** [singularityhub/sregistry#266]
**Describe the bug**

Upgrade of djangorestframework to latest release,  `3.11.0`, break `sregistry`, 
with bellow output errors:

```
TypeError: register() got an unexpected keyword argument 'base_name'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
  File "<frozen importlib._bootstrap_external>", line 697, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "./shub/urls.py", line 17, in <module>
    from shub.apps.api import urls as api_urls
  File "./shub/apps/api/urls/__init__.py", line 2, in <module> 
    from .routers import urlpatterns as routers
  File "./shub/apps/api/urls/routers.py", line 21, in <module>
    router.register(r"^containers", ContainerViewSet, base_name="container")
TypeError: register() got an unexpected keyword argument 'base_name'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
```
**To Reproduce**

docker build -t quay.io/vanessa/sregistry .

**Expected behavior**

**How to fix it**
```
git am --signoff -k << 'EOF'
From 518436526256c1e15909525a03957f43147d14a9 Mon Sep 17 00:00:00 2001
From: Kwame Amedodji <kamedodji@yahoo.fr>
Date: Wed, 18 Dec 2019 08:26:31 +0100
Subject: fix issue with djangoframework release 3.11.0

---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 0add329..78db8d7 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -19,7 +19,7 @@ django-taggit
 django-taggit-templatetags
 django-user-agents
 django>=1.11.18
-djangorestframework
+djangorestframework<3.11.0
 google
 google-api-python-client
 h5py
-- 
1.8.3.1
EOF

```
![sregistry_issue_latest_djangoframework_release](https://user-images.githubusercontent.com/21355617/71064508-87b39080-216f-11ea-96da-229285f5c882.png)

**Comment by vsoch** [comment]
The issue is that we need to pin the version of Django - what version do you have in that image?
```
pip freeze | grep Django
```

**Comment by vsoch** [comment]
See https://github.com/singularityhub/sregistry/pull/267

**Comment by kamedodji** [comment]
Here you are:
```
pip freeze | grep Django
Django==2.2.8
```

**Comment by vsoch** [comment]
Thanks! Wanted to check that too.

Could you please test the PR branch and let me know if it fixes the issue?

**Comment by kamedodji** [comment]
Yes i confirm @vsoch : is work fine right now!
Thx!

**Comment by vsoch** [comment]
Great! I’ll merge and go back to sleep then:)

**Comment by vsoch** [comment]
This was fixed with #267
